src/constants.js

```diff
@@ -246,3 +246,7 @@ export const HTTP_REQUEST_TIMEOUT = 3000;
  * Any network that is not mainnet or testnet should be a privatenet.
  */
 export const NETWORK_PRIVATENET = 'privatenet';
+
```

Comment:
Maybe add a comment above that the 3 constants below belong to the retry mechanism of sagas method calls. What do you think?

Reply:
Done at https://github.com/HathorNetwork/hathor-wallet-mobile/pull/444/commits/d5af3d6c10af31743d2deb82b4fbb7289201851c